hari/templates/404.html

```diff
@@ -0,0 +1,10 @@
+<!-- wp:template-part {"slug":"header-minimal","tagName":"header"} /-->
```

Comment:
to avoid duplicating templates just to add the different header, we changed Blockbase so all templates include the header.html template part, and inside that template part is where you insert the pattern you prefer, like `<!-- wp:pattern {"slug":"blockbase/header-minimal"} /-->`. Here's an example: the [index.html template](https://github.com/Automattic/themes/blob/f554716f193228fdbb8d7ee46c92a75280109b3f/videomaker/block-templates/index.html#L1) calls the header template part and the [header.html part](https://github.com/Automattic/themes/blob/trunk/videomaker/block-template-parts/header.html) includes the correct block pattern.

That way we only include templates that actually have changes on them, you don't need to copy over the 404, search, blank template, etc unless you want to modify further how they look.

Reply:
Updated.  New Blockbase variations and derivatives should follow that design.